test_scripts/API/DeleteCommand/ATF_DeleteCommand_UNSUP_RES_to_VR_DelCmd_WARNINGS_to_UI_DelCmd.lua

```diff
@@ -0,0 +1,167 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one component of RPC
+--
+-- Description:
+-- test is intended to check that SDL sends WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions
+-- in this particular test it is checked case when VR.DeleteCommand gets UNSUPPORTED_RESOURCE and UI.DeleteCommand gets WARNINGS from HMI
+-- 1. Used preconditions: App is registered and activated SUCCESSFULLY. AddCommand is sent SUCCESSFULLY
+-- 2. Performed steps: 
+-- MOB -> SDL: sends DeleteCommand
+-- SDL -> HMI: resends VR.DeleteCommand
+-- HMI -> SDL: VR.DeleteCommand (UNSUPPORTED_RESOURCE), UI.DeleteCommand (WARNINGS)
+--
+-- Expected result:
+-- SDL -> MOB: DeleteCommand(resultcode: UNSUPPORTED_RESOURCE, success: true)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.SDLStoragePath = config.pathToSDL .. "storage/"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+
+--[[ Local Variables ]]
+local storagePath = config.SDLStoragePath..config.application1.registerAppInterfaceParams.appID.. "_" .. config.deviceMAC.."/"
+local ServerAddress = commonFunctions:read_parameter_from_smart_device_link_ini("ServerAddress")
+
+--[[ General Precondition before ATF start ]]
+commonFunctions:SDLForceStop()
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
```

Comment:
@istoimenova , script was updated in commit  d2baa99d69379db10009b64a0949b062a9e6933f